lib/jekyll/tags/webmention.rb

```diff
@@ -16,6 +16,12 @@ class WebmentionTag < Liquid::Tag
       def initialize(tag_name, text, tokens)
         super
         cache_file = WebmentionIO.get_cache_file_path "incoming"
+
+        if Jekyll.sites.first.config['serving'] || Jekyll.sites.first.config['url'].include?('localhost') #Jekyll.env == 'development'
```

Comment:
Would this be

1. the right spot for those conditions? And
2. would those conditions be the correct ones at all?

As you can see, I also tried using `Jekyll.env` but this seemed not the right choice, since the `jekyll build` command uses the `development`environment by default, too. (And not sure about anyone else, but when I built my site manually, I always used to simply run `jekyll build` and then sync the `_site` fold via FTP. And if I’d be using `if Jekyll.env == ‘development’` here, this workflow would break.